lib/krane/cli/deploy_command.rb

```diff
@@ -16,6 +16,8 @@ class DeployCommand
         "filenames" => { type: :array, banner: 'config/deploy/production config/deploy/my-extra-resource.yml',
                          aliases: :f, required: true,
                          desc: "Directories and files that contains the configuration to apply" },
+        "std-in" => { type: :boolean, default: false,
```

Comment:
my preference is for the flag to be `--stdin` but I don't mind if I get outvoted

Reply:
I'll change to use `--stdin`, since https://en.wikipedia.org/wiki/Standard_streams